gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -237,6 +237,9 @@ func (vm *VMKeeper) AddPackage(ctx sdk.Context, msg MsgAddPackage) (err error) {
 // Call calls a public Gno function (for delivertx).
 func (vm *VMKeeper) Call(ctx sdk.Context, msg MsgCall) (res string, err error) {
 	pkgPath := msg.PkgPath // to import
+	if !gno.IsRealmPath(pkgPath) {
+		return "", errors.New("pkgpath must be a realm")
+	}
```

Comment:
Actually, move this to `vm/msgs.go`, in MsgCall.ValidateBasic. ValidateBasic is called earlier from the node, allowing to reject invalid msgcalls very quickly.

Please add a txtar test trying to do MsgCall against a package, and against a standard library.

Reply:
There are also tests failing in gno.land. These were added by thomas a short while ago; I think you can just remove the cases where the code was calling a package/stdlib.

Reply:
@thehowl hmm I moved it 🧔 
I also add new txtar test, can you give a look on this?